src/main/java/longah/LongAh.java

```diff
@@ -46,6 +49,13 @@ public static void main(String[] args) {
         LongAhLogger.log(Level.INFO, "Starting Pre-program preparations.");
         System.out.println("Welcome to LongAh!");
         LongAh app = new LongAh();
+        PINHandler pinHandler = new PINHandler();
+
+        if (!Files.exists(Paths.get(PINHandler.getPinFilePath()))|| pinHandler.loadPin().isEmpty()) {
```

Comment:
possible to have this done as part of pinHandler constructor? I'm trying to reduce clutter in main code at this point